Packs/ExtraHop/pack_metadata.json

```
@@ -2,11 +2,12 @@
     "name": "ExtraHop Reveal(x)",
     "description": "Network detection and response. Complete visibility of network communications at enterprise scale, real-time threat detections backed by machine learning, and guided investigation workflows that simplify response.",
     "support": "partner",
-    "currentVersion": "2.2.11",
+    "currentVersion": "2.2.12",
```

```suggestion
    "currentVersion": "2.3.0",
```
Don't forget to change the release notes file name as well